src/main.c

```diff
@@ -1879,15 +1879,15 @@ int main(int argc, char **argv)
     glBindTexture(GL_TEXTURE_2D, texture);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
-    load_png_texture("textures/texture.png");
+    load_png_texture("../textures/texture.png");
```

Comment:
This pathing makes more sense to me, but it isn't working when I run it in my environment. It was working with the other path. The error is: `load_png_texture ../textures/texture.png failed, error 78: failed to open file for reading`
Maybe there's a more universal solution?